syntax/index.html

```diff
@@ -649,14 +651,20 @@
             Split the header and data rows into cells using the <a>delimiter</a>. Values that are enclosed within the <a>quote character</a> may contain the <a>delimiter</a>. The <a>quote character</a> may be escaped using the <a>escape character</a> where it appears within cells. If the <a>escape character</a> is not the same as the <a>quote character</a> then the <a>escape character</a> is also used to escape the character that immediately follows it.
           </p>
           <p>
-            If <a>trim</a> is <code>true</code> or <code>start</code> then whitespace from the start of values that are not enclosed must be removed from the value. If <a>trim</a> is <code>true</code> or <code>end</code> then whitespace from the end of values that are not enclosed must be removed from the value.
+            If <a>trim</a> is <code>true</code> or <code>start</code> then whitespace from the start of values that are not enclosed MUST be removed from the value. If <a>trim</a> is <code>true</code> or <code>end</code> then whitespace from the end of values that are not enclosed MUST be removed from the value.
```

Comment:
Yes, the sentences should probably be re-written to sound less normative. Is there a place for normative text when processing the CSV? It would be useful to be able to refer to such steps normatively. The `trim` discussion could be moved to the Metadata description of `trim`.